src/types/observablemap.ts

```diff
@@ -294,7 +296,7 @@ export class ObservableMap<K = any, V = any>
     }
 
     forEach(callback: (value: V, key: K, object: Map<K, V>) => void, thisArg?) {
-        this._keys.forEach(key => callback.call(thisArg, this.get(key), key, this))
+        for (const [key, value] of this) callback.call(thisArg, value, key, this)
```

Comment:
Let's double check that this results in sane code for ES5, I think I avoided using `for..of` loops in this code base, due to issues in the past on IE 11, not entirely sure about that anymore